server/controllers/devices.coffee

```diff
@@ -61,7 +61,7 @@ deviceExists = (login, cb) ->
         else if body.length is 0
             cb null, false
         else
-            cb null, body[0]
+            cb null, body[0]?.value
```

Comment:
No idea, I checked the DS & Cradle too and couldn't find anything. 
Was this caught by travis ?